core/storage/activity/gae_models.py

```diff
@@ -49,7 +49,7 @@ def get_model_association_to_user():
 
     @classmethod
     def get_export_policy(cls):
-        """Model does not contain user data."""
+        """Model doesn't contain any data directly corresponding to a user."""
```

Comment:
Should the docstring in the `get_model_association_to_user` method also be updated?

Reply:
ditto elsewhere

Reply:
Done